docs/manufacturer/cloud-connector/index.md

```diff
@@ -56,8 +54,8 @@ Unzip the archive on your machine and follow the step. You can also find those s
 Before starting the installation make sure to prepare a CloudConnector configuration for the target machine to have all OPC UA server settings in place.
 
 ```console
-user@example:~$ tar xzvf Tapio.CloudConnector.<platform>.tar.gz
-user@example:~$ cd Tapio.CloudConnector.Service.Core
```

Comment:
würde hier mal Jim kurz fragen ob das noch passt. Rr musste zumindest eine .dll wieder von `Service` zu `Service.Core` umbennen. Denke aber das der Folder mit `Service` passen sollte.

Reply:
Folder with service is alright. Just double checked it.